### PR TITLE
Updated documentation and examples

### PR DESCRIPTION
**Pull request by Jemereny**
Updated userGuide for new features
Updated AddCommand's example to include website
Updated StatusBarFooter to show persons when starting app (before it shows only after the address book does something to a person)
Updated EditCommandParserTest to include website

### CODE REVIEW

File: docs/UserGuide.adoc [hunk 1]
```diff
@@ -60,19 +65,23 @@ Format: `help`
 === Adding a person: `add` / `a`
 
 Adds a person to the address book +
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS b/Birthday(DD/MM/YYYY) [t/TAG]...`
-Shorthand: `a n/NAME [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS b/Birthday(DD/MM/YYYY) [w/WEBSITE] [t/TAG]...`
```
Comment (xhsien) [inline docs/UserGuide.adoc]: At the very end you need to add a `+` for newline.

File: docs/UserGuide.adoc [hunk 2]
```diff
@@ -83,8 +92,8 @@ Shorthand: `l`
 === Editing a person : `edit` / `e`
 
 Edits an existing person in the address book. +
-Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [b/Birthday(DD/MM/YYYY)] [t/TAG]...`
```
Comment (xhsien) [inline docs/UserGuide.adoc:86]: Same as above. New line issue.

File: docs/UserGuide.adoc (hunk 1 above)
Comment (chilipadiboy) [reply docs/UserGuide.adoc]: Our book is Death Note or DeathNote?

Comment (Jemereny) [reply docs/UserGuide.adoc]: Missed it by accident. Updated.

**Comment by xhsien** [comment]
All comments have been addressed and fixed. I will merge this pr.

**Pull request merged**